### DISCUSSION TITLE
Dynamic VSelect - Populate VSelect with items dynamically

### DISCUSSION THREAD

**Discussion by juanprietob** [Kitware/trame#491] [Q&A] [answered]
The function LoadStudy is called and it updates the state variable 'study_images'. The function 'study_images_update' checks on the state change and is called as expected. 
```
def LoadStudy(study_id):
    #When this is updated, the function 'study_images_update' is called and it prints the updated state variable 'study_images'
    state.study_images = vc.LoadStudy(study_id)

@state.change("study_images")
def study_images_update(study_images, **kwargs):
    print(f"Updating {study_images}")
```

In my view, I have defined the VSelect component as follows:
```
with vuetify.VToolbar():
    vuetify.VSelect(items=state.study_images, v_model="study_img", label="Study Images")
```

However, the VSelect component does not update after the state variable study_images is updated. 
How can I dynamically populate a VSelect component?

**Comment by jourdain** [comment, marked as answer]
Please look at the [shared-state documentation](https://kitware.github.io/trame/guide/intro/getting_started.html#shared-state).

The solution is `vuetify.VSelect(items=("study_images", []), ...`
